apps/console/src/views/developer.tsx

```diff
@@ -194,6 +176,18 @@ export const DeveloperView: FunctionComponent<DeveloperViewPropsInterface> = (
 
         // Filter the routes and get only the enabled routes defined in the app config.
         setFilteredRoutes(routes);
+        setSelectedRoute(CommonRouteUtils.getInitialActiveRoute(location.pathname, routes));
+
+        const sanitizedManageRoutes: RouteInterface[] = CommonRouteUtils.sanitizeForUI(cloneDeep(manageRoutes));
+
+        const tab: string = AccessControlUtils.getDisabledTab(
+            sanitizedManageRoutes, routes, allowedScopes, featureConfig, commonConfig.checkForUIResourceScopes);
+
+        if (tab === "MANAGE") {
```

Comment:
Shall we get this from `AppViewTypes` enum.
```suggestion
        if (tab === "MANAGE") {
```